src/radical/pilot/bootstrapper/default_bootstrapper.sh

```diff
@@ -1414,6 +1414,22 @@ profile_event 'cleanup done'
 echo "#"
 echo "# -------------------------------------------------------------------"
 
+echo
+echo "# -------------------------------------------------------------------"
+echo "#"
+echo "# Tarring profiles ..."
+PROFILES_TARBALL="profiles.tgz"
+tar -czf $PROFILES_TARBALL *.prof
```

Comment:
Yes, exactly :) `tar zcf file.tgz ` an `tar zfc file.tgz` are equivalent.  OTOH, `tar -zcf file.tgz` and `tar -zfc file.tgz` are not.  But as said the line is correct, and the un-dashed version has one of the wierdest arg conventions, ever... :P